.github/workflows/ci.yaml

```
@@ -497,12 +497,14 @@ jobs:
             # Create a temp dir on the R: ramdisk drive for Windows. The default
             # C: drive is extremely slow: https://github.com/actions/runner-images/issues/8755
             mkdir -p "R:/temp/embedded-pg"
-            go run scripts/embedded-pg/main.go -path "R:/temp/embedded-pg"
+            mkdir -p "C:/temp/embedded-pg-cache"
```

the C drive is super slow as the comment above says

```suggestion
            mkdir -p "R:/temp/embedded-pg-cache"
```